frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -515,7 +531,8 @@ export default class FirebaseInteractor {
       | undefined = undefined;
 
     if (assessmentForSession !== undefined) {
-      assessmentDuration = assessmentForSession.data().durationInSeconds;
+      assessmentDuration =
+        Math.round(assessmentForSession.data().durationsInSeconds * 100) / 100;
```

Comment:
what's this change for / is the durationInSeconds not giving back a clean number or something?

Reply:
it's a float so this is just rounding to 2 decimal places so they don't get data that looks like 1254.12000000001 seconds